packages/react-next/src/Styling.ts

```diff
@@ -1 +1,3 @@
 export * from 'office-ui-fabric-react/lib/Styling';
+
+export * from './Styling/ThemeProvider';
```

Comment:
Hmm. Should this be in `components/ThemeProvider`?

We should probably just avoid having folders that are named the same as files which can be imported.

Reply:
What's the downside of having a folder with same name? we do this today in OUFR. also I think `ThemeProvider` fits better in `lib/Styling`